packages/codemods/.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - 'v*' # older version branches
+    tags:
+      - '*'
+  pull_request: {}
+  schedule:
+  - cron:  '0 6 * * 0' # weekly, on sundays
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: install dependencies
+      run: yarn install --frozen-lockfile
+    - name: linting
+      run: yarn lint
+
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node: ['10', '12', '14']
```

Comment:
What are we testing here? 🤔

Reply:
This was the more or less auto-generated CI config. I would lean towards disabling these jobs and only running the tests locally when writing the code mods. I think the overhead is quite big and don't think it's worth running them in CI.